core/src/main/java/io/grpc/ManagedChannelProvider.java

```diff
@@ -125,7 +125,8 @@ public static ManagedChannelProvider provider() {
    */
   protected static boolean isAndroid() {
     try {
-      Class.forName("android.app.Application", /*initialize=*/ false, null);
+      // Specify a class loader instead of null because we may be running under Robolectric
+      Class.forName("android.app.Application", /*initialize=*/ false, ManagedChannelProvider.class);
```

Comment:
Yeah, done. I manually copied a small part of an internal CL, and obviously I copied it incompletely.